compliance_checker/tests/test_cf.py

```diff
@@ -177,10 +177,8 @@ def test_check_dimension_order(self):
         """
         dataset = self.load_dataset(STATIC_FILES['bad_data_type'])
         result = self.cf.check_dimension_order(dataset)
-        assert result.value == (5, 7)
-        assert result.msgs[0] == ("salinity's dimensions are not in the recommended order "
-                                  "T, Z, Y, X. They are latitude, longitude, time")
-        assert result.msgs[1] == ("really_bad's dimensions are not in the recommended order "
+        assert result.value == (5, 6)
```

Comment:
These lines had to be changed due to the new bounds order checking.  Checks against any bounds variables are deferred until the `check_cell_boundaries` test.  It would likely also fail on account of the fact it has a one-dimensional bounds variable with only one instead of two elements.